test/azure/generated/AzureReport/autoRestReportServiceForAzure.ts

```diff
@@ -17,10 +18,15 @@ import { AutoRestReportServiceForAzureContext } from "./autoRestReportServiceFor
 class AutoRestReportServiceForAzure extends AutoRestReportServiceForAzureContext {
   /**
    * Initializes a new instance of the AutoRestReportServiceForAzure class.
-   * @param credentials Credentials needed for the client to connect to Azure.
+   * @param credentials Credentials needed for the client to connect to Azure. The simplest
+   * TokenCredential credential can be obtained as follows:
```

Comment:
Can you please add a link to Identity's documentation so that they can learn about the ENV variables that need to be setup and more info?

Reply:
Fixed.